core/setup.py

```diff
@@ -48,6 +48,7 @@
     install_requires=[
         "Jinja2==3.1.2",
         "agate>=1.6,<1.6.4",
+        "betterproto>=1.2.5",
```

Comment:
Do we have a standard on how we're pinning packages?  Most (but not all) the time we seem to either pin to an exact version or a range with lower and upper bounds.

Reply:
I don't know if there's a standard. Do you think we should put == instead?

Reply:
Our policies for these things are WIP ([here](https://www.notion.so/WIP-Core-Dependency-Policies-483df216b3c64c989c0720a766db6f03)), but basically: IMO we shouldn't tightly pin (`==`) if we don't need. (It makes sense to do that in "locked" requirements files that guarantee a working install, but not for setup requirements.) And the reason to add an upper bound is just to avoid breaking changes that could come in v2.0.0. Have we tested with older versions of betterproto v1, before 1.2.5? If it can be as simple as `betterproto>1,<2`, amazing. If we're using capabilities that are new since v1.0, then `betterproto>=1.2.5,<2` should do the trick.

Reply:
v1.2.5 was released May 2020.  Looks like they've been actively working on a beta 2.0 since then.  Since the release is older, I would vote to pin it.  Seems unlikely they will release any new minor/patch releases for v1.  I can't find any timelines for the beta but have an open question in their slack about it.  @gshank were you able to find anything about timelines?

Reply:
As I recall the betterproto 2.0 beta has some great features but definitely some breaking/unexpected behavior. Not locking to the major version seems risky

Reply:
also there can be unexpected behavior if we compile with a different betterproto version than we use at runtime

Reply:
Pinned to 1.2.5